.github/workflows/pr-commit.yml

```diff
@@ -0,0 +1,46 @@
+name: PR Commit
+on:
+  pull_request:
+    branches: [main]
+jobs:
+  prettier-format:
+    name: Prettier Format
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout out repository code
+        uses: actions/checkout@v2
```

Comment:
We can use @v3 here (I'm not sure if it's better, but it's newer!

Also, in Front Row we are setting the fetch depth to 0:
```yml

      - name: Checkout code into workspace directory
        uses: actions/checkout@v3
        with:
          fetch-depth: 0
```
I think it's because it is faster on a large repository, it doesn't grab all the historical data.

Reply:
Got it, thanks!